lumigator/python/mzai/sdk/tests/test_jobs.py

```diff
@@ -64,26 +64,26 @@ def test_get_jobs_none(mock_requests_response, mock_requests, lumi_client):
     assert jobs.items == []
 
 
-def _test_get_job_ok(mock_requests_response, mock_requests, lumi_client, json_data_job):
+def test_get_job_ok(mock_requests_response, mock_requests, lumi_client, json_data_job):
     mock_requests_response.status_code = 200
     data = load_json(json_data_job)
     mock_requests_response.json = lambda: data
 
     job_id = "6f6487ac-7170-4a11-af7a-0f6db1ec9a74"
     job = lumi_client.health.get_job(job_id)
 
-    # Test some properties
     assert job is not None
     assert job.type == "SUBMISSION"
+    assert job.status == "PENDING"
     assert job.submission_id == job_id
 
 
-def _test_get_job_id_does_not_exist(mock_requests_response, mock_requests, lumi_client):
+def test_get_job_id_does_not_exist(mock_requests_response, mock_requests, lumi_client):
     mock_requests_response.status_code = HTTPStatus.NOT_FOUND
     mock_requests_response.json = lambda: None
     error = HTTPError(response=mock_requests_response)
     mock_requests.side_effect = error
 
     # We expect the SDK to handle the 404 and return None.
-    job = lumi_client.health.get_job("6f6487ac-7170-4a11-af7a-0f6db1ec9a74")
+    job = lumi_client.health.get_job("5f0487ac-7170-4a11-af7a-0f6db1ec9a74")
```

Comment:
```suggestion
    job = lumi_client.health.get_job("00000000-0000-0000-0000-000000000000")
```
This seems to be a special UUID instance as per https://datatracker.ietf.org/doc/html/rfc4122#section-4.1.7 and I would assume that most systems will not assign it as a valid UUID.